src/routes/games/__error.svelte

```diff
@@ -0,0 +1,28 @@
+<script context="module" lang="ts">
+	export function load({ error, status }) {
+		if (error.message !== 'offline') {
+			return {
+				error,
+				status
+			};
+		} else {
+			return {};
+		}
+	}
+</script>
+
+<header>
+	<h1>Spel niet offline beschikbaar</h1>
+</header>
+<main>
+	<p>Dit spel is niet gedownload en dus niet offline beschikbaar</p>
```

Comment:
```suggestion
	<p>Dit spel is niet gedownload en dus niet offline beschikbaar.</p>
```

😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 

🐜 🍆 🏃